tools/mdfmt/BUILD.bazel

```diff
@@ -1,12 +1,7 @@
-load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
 load("//bazel/utils:diff_test.bzl", "diff_test")
 load("//tools/mdfmt:mdfmt.bzl", "mdfmt_filter")
 load("@enkit_pip_deps//:requirements.bzl", "requirement")
 
-bzl_library(
-    name = "mdfmt_bzl",
```

Comment:
What's the purpose for removing this?

Reply:
It's broken and unused, and the new version of bazel_skylib complains about a bzl_library with no deps or sources.